src/main/groovy/com/ca/apim/gateway/modularassertionbuilder/ModularAssertionBuilder.groovy

```diff
@@ -79,7 +79,7 @@ class ModularAssertionBuilder implements Plugin<Project> {
                 }
 
                 assertionClasses = getAssertionClasses(
-                        project.sourceSets.main.output.classesDirs,
+                        project.sourceSets.main.output.classesDirs + project.sourceSets.main.output.resourcesDir,
```

Comment:
you can just concatenate the two dirs like this? no need for some delimiter?

Reply:
yea this is fine. there is existing code like this already, see line 95